draft-ecahc-moderation.md

```diff
@@ -189,11 +189,26 @@ mailing lists, chat channels, and discussions in other systems that
 the IETF or WGs have chosen to employ, such as GitHub repositories,
 Wikis, or issue trackers.
 
-The management and moderation of in-person, remote, and interim
-meetings remains a task of the relevant group's management, such as
-WG chairs. However, it is expected that moderators are available for
-consultation and assistance should issues arise, and they may confer
-with the group management over potential patterns of behavior.
+It is important to note that the moderation team only
+moderates *public* IETF participation channels. Their mandate does
+not extend to problematic behavior in private channels, such as
+private chat channels, direct messages, or conversations or other
+interactions outside of meetings. In any such cases, the Ombudsteam
+should be approached.
+
+The management and moderation of participation channels associated
+with various IETF groups, inculding their mailing lists, chat
+channels and in-person, remote, or interim meetings SHOULD remain a
+task of the relevant group's management, such as WG chairs. However,
+it is expected that moderators are available for consultation and
+assistance should issues arise, and they may confer with the group
+management over potential patterns of behavior. When moderators
```

Comment:
Editorial: over use of passive voice. "it is expected" is too much.

Can this concern be phrased more directly?

Reply:
wfm